src/pages/DetailPage/components/ProductInquiry/ProductInquiry.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import '../ProductInquiry/ProductInquiry.scss';
+import InquiryModal from './InquiryModal/InquiryModal';
+
+function ProductInquiry() {
+  const [modal, setModal] = useState(false);
+
+  const handleButtonClick = () => {
+    setModal(true);
+  };
+
+  const handleXClick = () => {
+    setModal(false);
+  };
+
+  return (
+    <div className="productInquiry">
+      <div className="description">
+        상품에 대한 궁금한 점을 남겨주시면 판매자가 직접 확인 후 답변을
+        드립니다.
+      </div>
+      <button className="inquiryButton" onClick={handleButtonClick}>
+        상품문의 남기기
+      </button>
+      {modal === true ? <InquiryModal handleX={handleXClick} /> : null}
```

Comment:
```suggestion
      {modal && <InquiryModal handleX={handleXClick} /> }
```
modal이 true 일 때만 render 되도록 단축해서 사용할 수 있습니다.